api/v2/routes_frontend_test.go

```diff
@@ -32,7 +32,7 @@ func Test_API_Routes_Frontend(t *testing.T) {
 	fakeSigner := &mocks.FakeSignerClient{}
 	fakeWalletService := &mocks.FakeWalletServiceClient{}
 
-	api, testRecorder, err := setupAPI(t, fakeLens, fakeOrch, fakeSigner, fakeWalletService, cfg, db)
+	api, _, err := setupAPI(t, fakeLens, fakeOrch, fakeSigner, fakeWalletService, cfg, db)
```

Comment:
I think the return of testRecorder should be removed from setupAPI. It is never acturally used in any test code.

Reply:
Good idea